packages/js/plugins/ethereum/src/index.ts

```diff
@@ -154,7 +154,7 @@ export class EthereumPlugin extends Plugin {
     const network = await provider.getNetwork();
     return {
       name: network.name,
-      chainId: network.chainId,
+      chainId: network.chainId.toString(),
```

Comment:
Aren't we serializing this automatically?

Reply:
That would be true if this were a wrapper, but in plugins the `BigInt` type is an alias for `string`. Here I'm changing the type from `number` to `string`.